### DISCUSSION TITLE
zh/guide/drivers/weiyun

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#251] [Announcements]
# zh/guide/drivers/weiyun

腾讯微云官网：https://www.weiyun.com 微云登录有效期提示 QQ：登录目前只能保活48小时就得更换cookie; 微信：无此问题，用的是token; 推荐使用微信登录 根文件夹ID 1. 展示根目录，留空 就可以不用填写，程序会自动填充 2. 如果只展示其中单个文件夹ID，进入想展示的文件夹內然后复制顶部地址栏后面的ID Cooki...

https://alist.nn.ci/zh/guide/drivers/weiyun.html

**Comment by seawave0564** [comment]
保存一直转圈圈

**Comment by simplove** [comment]
下载速度太慢，只有几十KB，有啥用，免费用户也只有10G空间

**Reply by meowhuan** [reply]
123云盘的下载速度慢吗？他们免费用户不是有两TB吗？要我说你说出这句话，我只能说你没用过（）

**Reply by meowhuan** [reply]
额，抱歉（）浏览器问题，我以为点到了123云盘那（抱歉

**Comment by meowhuan** [comment]
- 123云盘有open API了
> https://123yunpan.yuque.com/org-wiki-123yunpan-muaork/cr6ced

### 我之所以提出这个
就是希望alist官方能够支持这个open API 我也不强求
就单纯希望（）

**Comment by spz2020** [comment]
有人知道微信如何扫自己手机上的登录二维码吗？

**Reply by lostdv10** [reply]
分屏，截屏

**Comment by foxdaiwx** [comment]
视频如何播放？

**Comment by lizhiwei8061** [comment]
{"code":500,"message":"failed link: failed get link: cmd:2402, cmdName:DiskFileBatchDownload, seq:870241536, errcode:190054, errmsg:(190054)请求次数超限，请等待一段时间后重试","data":null} 请问这个多久解除
